### ISSUE TITLE
Getting started issue

### ISSUE THREAD

**Issue by aadityarajkumawat** [sullivan-sean/chat-langchainjs#6]
```bash
# yarn ingest is not working
yarn ingest
```

<img width="1062" alt="Screenshot 2023-02-21 at 7 48 06 PM" src="https://user-images.githubusercontent.com/51184441/220369987-1bceaccd-0754-46bc-973d-7105f39d7d01.png">

**Comment by sambroe** [comment]
I'm experiencing the same thing. When I follow the suggestion to add `"type":"module"` I get a TypeError `TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for ingest.ts`

**Comment by aadityarajkumawat** [comment]
Yes, I get the same error, I really want to use this, but not sure how to get this resolved.

**Comment by hwchase17** [comment]
can you try the instructions now?

**Comment by jackson-sandland** [comment]
Related to getting started. I've created a `.env` file and added my API_KEY but I don't see a reference to it anywhere and I'm getting this error: 
```
    data: {
      error: {
        message: 'Incorrect API key provided: undefined. You can find your API key at https://platform.openai.com/account/api-keys.',
        type: 'invalid_request_error',
        param: null,
        code: 'invalid_api_key'
      }
    }
```

**Comment by aadityarajkumawat** [comment]
I am looking forward to seeing you today for our meeting. I’ve attached the materials needed for our discussion. Let me know if you have any questions.
